### ISSUE TITLE
Flake8 rule 'E901' reported an invalid point

### ISSUE THREAD

**Issue by michaelaye** [AtomLinter/linter-flake8#311]
Flake8 reported an invalid point for the rule `E901`, with the messge `SyntaxError: invalid syntax`.

This is the beginning of my module:

```python
"""Managing clustering, fnotching and cut application here."""
from __future__ import division, print_function

import importlib
import logging
from pathlib import Path

import matplotlib
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
from numpy.linalg import norm
from scipy.stats import circmean

from . import io, markings
from .dbscan import DBScanner, HDBScanner

importlib.reload(logging)
logpath = Path.home() / 'p4reduction.log'
logging.basicConfig(filename=str(logpath), filemode='w', level=logging.DEBUG,
                    format='%(asctime)s %(levelname)s: %(message)s',
                    datefmt='%Y-%m-%d %H:%M:%S')

matplotlib.style.use('bmh')

```
The error was shown at line 1, pos 1, as shown in screenshot.

<img width="715" alt="screenshot 2016-12-14 17 26 29" src="https://cloud.githubusercontent.com/assets/69774/21206975/9a904eb4-c222-11e6-9f8e-4fcc8b1c3c49.png">

Debug information:
Atom version: 1.12.7
Flake8 version: `2.5.1 (pep8: 1.7.0, pyflakes: 1.3.0, mccabe: 0.3.1) CPython 3.5.2 on Darwin`

**Comment by michaelaye** [comment]
I had a broken syntax much further done, inside a function:
```python
    def cluster_angles(self, filtered):
        # calculated value of euclidean distance of unit vector end points per degree
        dist_per_degree = 0.017453070996747883
        X =
```
Maybe related?

**Comment by Arcanemagus** [comment]
Please update your `flake8` version (`pip install -U flake8`). This particular error should be fixed in `flake8>=3.2.1`.

**Comment by michaelaye** [comment]
Thank you! conda's default channel is still on 2.x that's how I came to that. Will ask them why they don't upgrade.
